app/misc/git.ts

```diff
@@ -495,6 +495,37 @@ function createBranch() {
   }
 }
 
+// search for tags
+function searchTag() {
+  Git.Repository.open(repoFullPath)
+    .then(function (repo) {
+      repo.getCurrentBranch()
+        .then(function () {
+          // grab the list of references - these could be branches or tags
+          return repo.getReferences(Git.Reference.TYPE.LISTALL);
+        }).then(function (refList) {
+          refList.sort();
+          for (let i = 0; i < refList.length; i++) {
+
+            // strip name for readability
+            let refName = refList[i].name().split("/")[refList[i].name().split("/").length - 1];
+
+            if (refList[i].isTag()){
+              if (refName.indexOf( document.getElementById("tag-name").value ) > -1) {
+                document.getElementById(refName).setAttribute("style","display:block");
+              } else {
+                document.getElementById(refName).setAttribute("style","display:none");
```

Comment:
lines 515 and 517 could be refactored?
if > -1
  attr = "display:block"
else
  attr = "display:none"
document.getelementbyid(refname).setattribute("style", attr)

Reply:
Sure, done.